src/utils.ts

```diff
@@ -1405,3 +1419,32 @@ export function parsePackageVersion({ version }: { version: string }): {
   const [major, minor, patch] = version.split('.').map((n: string) => Number.parseInt(n, 10));
   return { major, minor, patch };
 }
+
+/**
+ * Fisher–Yates Shuffle
+ *
+ * Reference: https://bost.ocks.org/mike/shuffle/
+ * @param sequence - items to be shuffled
+ * @param limit - if nonzero shuffle will slice the randomized array e.g, `.slice(0, limit)`
```

Comment:
add something along the lines of "otherwise will return the entire randomized array, same as in the case where the limit is not specified" ?